### DISCUSSION TITLE
Task going into an exclusion

### DISCUSSION THREAD

**Discussion by rafalkrupinski** [GothenburgBitFactory/timewarrior#518] [Q&A]
When a start a task and I let it go into exclusion time, the summary will show that task as stopped, even tho it isn't.
I find that confusing. Is that on purpose?
I thought exclusions only affect `track` command.

**Comment by lauft** [comment]
I guess this is on purpose. An exclusion defines a time period where you do not (want to) track your time. So without further input from your side, when reaching the exclusion Timewarrior can only assume that you stopped working and thus show the time tracking as stopped (it will continue after the exclusion then).

**Reply by rafalkrupinski** [reply]
Yeah, I find the whole concept confusing.
I like it's showing me visual time constraints in the time charts (day, week, month charts), but it's not obvious what happens with my tracked time.
if I start tracking time outside of exclusion and stop within, did it track all the time or just until the exclusion?
What happens when an exclusion falls within a tracked time session?
What if I want to track an emergency overnight work session?

I think the whole idea is more complex than I'd expect from a time tracker.

**Comment by paulinhoneto** [comment]


**Reply by lauft** [reply]
@paulinhoneto Timewarrior is a _time tracker_ not a task manager. Your proposals sound more like something for [Taskwarrior](https://github.com/GothenburgBitFactory/taskwarrior)... 🤔

**Reply by paulinhoneto** [reply]
Clear here
